src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -165,3 +166,17 @@ private sealed class NoOpDisposable : IDisposable
         public void Dispose() { }
     }
 }
+
+internal static partial class ConsoleLoggerInterop
+{
+    [JSImport("globalThis.console.debug")]
+    public static partial void ConsoleDebug(string message);
+    [JSImport("globalThis.console.info")]
+    public static partial void ConsoleInfo(string message);
+    [JSImport("globalThis.console.warn")]
+    public static partial void ConsoleWarn(string message);
+    [JSImport("globalThis.console.error")]
+    public static partial void ConsoleError(string message);
+    [JSImport("Blazor._internal.dotNetCriticalError")]
+    public static partial void DotNetCriticalError(string message);
+}
```

Comment:
It's great to have at least this one usage example so we will have a clear sense of how to extend this pattern throughout the Blazor sources in .NET 8.